frappe/email/doctype/notification/test_notification.py

```diff
@@ -115,10 +116,10 @@ def test_alert_disabled_on_wrong_field(self):
 		event.subject = "test 1"
 		event.save()
 
-		# verify that email_alert is disabled
-		email_alert.reload()
-		self.assertEqual(email_alert.enabled, 0)
-		email_alert.delete()
+		# verify that notification is disabled
+		notification.reload()
+		self.assertEqual(notification.enabled, 0)
+		notification.delete()
```

Comment:
add `delete` to the other tests too. Maybe this test fails because of previous tests (the sequence of unit - tests is random)

Reply:
Hi @rmehta,

I have also added `delete` in the other test creating a notification and have tried desperately to reproduce Travis' error but with no luck so far... :disappointed: 
On a local site it works fine when running `bench run-tests --doctype Notification`